scripts/global.js

```diff
@@ -475,25 +463,55 @@ export async function updateStakingRewardsGUI() {
     }
 
     //DOMS.DOM-optimised list generation
-    const strList = arrRewards
-        .map(
-            (cReward) =>
-                `<i style="opacity: 0.75; cursor: pointer" onclick="window.open('${
-                    cExplorer.url + '/tx/' + cReward.id
-                }', '_blank')">${new Date(
-                    cReward.time * 1000
-                ).toLocaleDateString()}</i> <b>+${cReward.amount} ${
-                    cChainParams.current.TICKER
-                }</b>`
-        )
-        .join('<br>');
+    let strList = `
+    <table class="table table-responsive table-sm stakingTx table-mobile-scroll">
+        <thead>
+            <tr>
+                <th scope="col" class="tx1">Time</th>
+                <th scope="col" class="tx2">Hash</th>
+                <th scope="col" class="tx3">Amount</th>
+                <th scope="col" class="tx4 text-right"></th>
+            </tr>
+        </thead>
+        <tbody>`;
+
+        let i = true;
+    arrRewards.map(
+        (cReward) => {
+            let dateTime = new Date( cReward.time * 1000 );
```

Comment:
Could you refactor the date algorithm to a new function?

Reply:
I've done some "trimming" and it's quite small now, do you still think it should be shipped to a new function? I think it's now not worth it due to the tiny size.

Reply:
I think it's long enough, plus it will be reused in other places